engine/runner/contracts/FrontDoor.sol

```diff
@@ -5,17 +5,8 @@ import './Proxy.sol';
 import './LibStorageWriter.sol';
 
 contract FrontDoor is Proxy {
-  constructor() Proxy(msg.sender, address(new EternalStorage(msg.sender, address(0))), address(0x0), false) {
+  constructor() Proxy(address(new EternalStorage(msg.sender, address(0)))) {
     bytes32 key = keccak256(abi.encodePacked('frontDoor'));
     StorageWriter.setAddress(eternalStorageAddress, key, address(this));
```

Comment:
@0x-adam-smith what is the function of this? It delegatecalls eternalStorageAddress and **writes to the state of FrontDoor**. Test passes with or without this line. Is it a relic that can be removed?